docs/decisions/0005-project-labels.md

```diff
@@ -0,0 +1,152 @@
+# Context and Problem Statement
+
+The goal is to put into place standard labels across all SUSE Rancher projects,
+which allows people across multiple spheres of interest a single glance into
+top-level work.
+
+All projects under the SUSE Rancher org are defining standard labels, which will
+allow people across multiple domains of interest a single glance into top-level
+work.
```

Comment:
Is this repeating the same concept? 
Also, the first sentence sounds weird to me:
```
which allows people across multiple spheres of interest a single glance into top-level work
```

Reply:
Thanks for spotting it! It should be fixed now

Reply:
I also reworded the text a bit